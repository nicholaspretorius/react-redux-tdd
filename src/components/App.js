import React, { Component } from "react";

import Wallet from "./Wallet";

class App extends Component {
  state = {};
  render() {
    return (
      <div>
        <h3>Loot Check</h3>
        <Wallet />
      </div>
    );
  }
}

export default App;
