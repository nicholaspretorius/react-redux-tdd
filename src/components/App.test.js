import React from "react";
import { shallow } from "enzyme";

import App from "./App";

describe("App", () => {
  const app = shallow(<App />);

  it("render the app correctly", () => {
    expect(app).toMatchSnapshot();
  });

  it("contains a connected Wallet component", () => {
    // console.log(app.debug());
    expect(app.find("Connect(Wallet)").exists()).toBe(true);
  });
});
