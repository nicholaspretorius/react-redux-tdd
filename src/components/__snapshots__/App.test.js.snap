// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App render the app correctly 1`] = `ShallowWrapper {}`;
